Add Sage-specific options to the gap.sh source file

diff -ur old/gap.shi src/gap.shi
--- old/gap.shi	2012-02-20 20:04:01.000000000 +0000
+++ src/gap.shi	2012-09-02 10:44:27.488429081 +0100
@@ -8,7 +8,10 @@
 ##  Then copy this file to a  directory in your  search path,  e.g., '~/bin'.
 ##  If you later move GAP to another location you must only change this file.
 ##
-
+if [ $SAGE_LOCAL"" = "" ]; then
+    echo "Set the environment variable SAGE_LOCAL."
+    exit 1
+fi
 
 #############################################################################
 ##
@@ -20,7 +23,7 @@
 ##  You won't have to change this unless you move the installation.
 ##
 if [ "x$GAP_DIR" = "x" ];  then
-GAP_DIR="@gapdir@"
+    GAP_DIR="$SAGE_LOCAL/gap/latest"
 fi
 
 
@@ -35,9 +38,9 @@
 ##  in parallel with other programs you may want to set this value close
 ##  to the amount of memory your computer has.
 ##
-#if [ "x$GAP_MEM" = "x" ];  then
-#GAP_MEM="-m 256m"
-#fi
+if [ "x$GAP_MEM" = "x" ];  then
+    GAP_MEM="-m 24m"
+fi
 
 
 #############################################################################
